devbox.json

```diff
@@ -31,6 +31,7 @@
     "shfmt@latest",
     "yamale@latest",
     "yamllint@latest",
+    "yq-go@latest",
```

Comment:
Could just use `gojq` for this instead so we don't need another tool? `gojq` supports YAML input and output (with appropriate flags).

Reply:
Ah I see you're using it for in-place editing.